src/main/java/bio/terra/cli/command/shared/options/WorkspaceOverride.java

```diff
@@ -17,12 +17,12 @@ public class WorkspaceOverride {
   @CommandLine.Option(
       names = "--workspace",
       description = "Workspace id to use for this command only.")
-  private String userFacingId;
+  private String id;
```

Comment:
Let's add comment since this is easy to forget.

```
// Variable is `id` instead of `userFacingId` because user sees it when they run a command with missing required arguments
```